src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -100,7 +100,7 @@ def pull_helm_chart(registry_path, kube_config, kube_context):
     if response_helm_chart_pull.returncode != 0:
         telemetry.set_exception(exception=error_helm_chart_pull.decode("ascii"), fault_type=consts.Pull_HelmChart_Fault_Type,
                                 summary='Unable to pull helm chart from the registry')
-        raise CLIError("Unable to pull helm chart from the registry '{}': ".format(registry_path) + error_helm_chart_pull.decode("ascii"))
+        raise CLIInternalError("Unable to pull helm chart from the registry '{}': ".format(registry_path) + error_helm_chart_pull.decode("ascii"))
```

Comment:
Pls ensure that we are not re-calling telemetry.set_exception during error handling. For eg, such calls should not be made:
https://github.com/Azure/azure-cli/blob/119856d4f5984e5a4ef0cdb5bc233c6f07da2720/src/azure-cli-core/azure/cli/core/azclierror.py#L114

Reply:
We are not setting telemetry.set_exception_trace() explicitely, so it won't re-raise the exception.